framework/core/Constants.pm

```diff
@@ -161,17 +161,25 @@ The directory of the libraries of the build system tools (I<C<LIB_DIR>/build_sys
 =cut
 our $BUILD_SYSTEMS_LIB_DIR = ($ENV{'BUILD_SYSTEMS_LIB_DIR'} // "$LIB_DIR/build_systems");
 
-
 =pod
 
 =item C<D4J_BUILD_FILE>
 
 The top-level (ant) build file (I<C<SCRIPT_DIR>/projects/defects4j.build.xml>)
 
+=cut
+our $D4J_BUILD_FILE = ($ENV{'D4J_BUILD_FILE'} or "$SCRIPT_DIR/projects/defects4j.build.xml");
+
+=pod
+
+=item C<GRADLE_LOCAL_HOME_DIR>
+
+The directory name of the local gradle repository (.gradle_local_home).
+
 =back
 
 =cut
-our $D4J_BUILD_FILE = ($ENV{'D4J_BUILD_FILE'} or "$SCRIPT_DIR/projects/defects4j.build.xml");
+our $GRADLE_LOCAL_HOME_DIR = ".gradle_local_home";
```

Comment:
Is this relative to `framework/core`, the working directory of a checked-out project version, or something else?

Reply:
The working directory of a particular project version.